.github/workflows/ci.yaml

```diff
@@ -113,7 +114,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        datalossprevention: ["true", "false"]
+        ppnd: ["true", "false"]
```

Comment:
this more accurately reflects what we're doing - either testing with "ppnd" or no strategy